resources/asciidoctor/lib/edit_me/extension.rb

```diff
@@ -74,8 +74,16 @@ def link_for(block)
       url = edit_url block
       return '' unless url
 
+      if block.document.attr 'private_edit_urls'
+        css_class = 'edit_me edit_me_private'
```

Comment:
s/css_class/css_classes/ ?

Reply:
you already reviewed this last time :D , but sure, i'm fine with that too.

Reply:
:man_shrugging: Sorry. Either way is fine I guess.

Reply:
I'd use `css_class_string` or `css_classes` so changed to the plural.

Reply:
I'd use `css_class_string` or `css_classes` so changed to the plural.